src/site/resources/js/anchors.js

```diff
@@ -56,6 +56,25 @@
             anchor.appendChild(a);
             anchorItem.appendChild(anchor);
         });
+
+        var exampleDivs = document.querySelectorAll('p[id^="Example"]');
```

Comment:
I really have no knowledge of JS syntax/best practices, but is there some way to make these final? `var` -> `val` (by quick google search)? Or is this not generally done?

Reply:
We can do `const`. It's pretty common actually. I just noticed the other code not doing it and followed.

Reply:
Done. Changed all to `const`.